.github/ISSUE_TEMPLATE/question.md

```diff
@@ -8,6 +8,11 @@ labels: ❔Question
 
 <!-- Run `react-native info` in your terminal and paste its contents here. -->
 
+## Way of upgrading
```

Comment:
What about something like the following, which implicitly hints at “you should try this, at least” ?

```suggestion
## Things I’ve done to figure out my issue
```